Spanner/tests/Unit/DatabaseTest.php

```diff
@@ -131,7 +133,20 @@ public function setUp(): void
             'connection', 'operation', 'session', 'sessionPool', 'instance'
         ];
 
+        array_push($args, ['includeReplicas' => ['us-central1']]);
         $this->database = TestHelpers::stub(Database::class, $args, $props);
+        $this->databaseWithIncludeReplicas = TestHelpers::stub(
+            Database::class,
+            $args,
+            $props
+        );
+        $args[10] = ['excludeReplicas' => ['us-central1']];
+        $this->databaseWithExcludeReplicas = TestHelpers::stub(
+            Database::class,
+            $args,
+            $props
+        );
+        $args[10] = [];
         $args[6] = null;
```

Comment:
nit: unset the key, rather than the index. So that adding an argument in the `$args` array won't lead to a trickle down change for all indices.

Reply:
The problem is, it will be passed to a function as arguments, so we need to pass the required values like `null` or empty array `[]` in the `$args`. Thats why I added these lines. What do you think. Please let me know if you need to have a discussion on this.

Reply:
Changing the logic as `includeReplicas` and `excludeReplicas` are serialized solely in the Grpc.php